submissions/akinbobolaFaith/phpMySqlDatabase/customersList.php

```diff
@@ -0,0 +1,41 @@
+<?php
+require_once("cookie.php");
+
+if (isset($_COOKIE["customers"])) {
+    $data = json_decode($_COOKIE["customers"], true);
```

Comment:
What if the cookie is not set. What becomes of `$data`

Reply:
Data should be returned. I don't know how to write the code that returns the data. I used return $data

Reply:
Data should be returned. I don't know how to write the code that returns the data. I used return $data